code/game/machinery/doors/airlock.dm

```diff
@@ -650,6 +650,17 @@ About the new airlock wires panel:
 				visible_message("<span class='warning'>[user] headbutts the airlock. Good thing [user.p_theyre()] wearing a helmet.</span>")
 			return
 
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.a_intent==INTENT_GRAB)
```

Comment:
Do not nest conditionals when an `&&` will do its job.
Additionally, please leave spaces around this `==` statement.